README.md

```diff
@@ -101,6 +101,15 @@ export async function updateDocumentTitle(_id, title) {
     - [2. Cancel a request by unsubscribing from the Observable](#2-cancel-a-request-by-unsubscribing-from-the-observable)
   - [Get client configuration](#get-client-configuration)
   - [Set client configuration](#set-client-configuration)
+  - [Server Side Actions](#server-side-actions)
+    - [Create Action](#create-action)
+    - [ReplaceDraft Action](#replacedraft-action)
+    - [Edit Action](#edit-action)
+    - [Delete Action](#delete-action)
+    - [Discard Action](#discard-action)
+    - [Publish Action](#publish-action)
+    - [Unpublish Action](#unpublish-action)
+    - [Action options](#action-options)
```

Comment:
Please could you put these in alphabetical order to match the documentation? (https://www.sanity.io/docs/http-actions)

Reply:
Reordered items in order.